vault/resource_database_secret_backend_static_role.go

```diff
@@ -99,6 +101,20 @@ func databaseSecretBackendStaticRoleResource() *schema.Resource {
 				Description: "The password corresponding to the username in the database. " +
 					"Required when using the Rootless Password Rotation workflow for static roles.",
 			},
+			consts.FieldCredentialType: {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "password",
```

Comment:
Let's use computed so that TFVP can rely on the default set by Vault.
```suggestion
				Computed:  true,
```